.github/workflows/lint.yml

```diff
@@ -25,7 +25,13 @@ jobs:
           pip install -r requirements.txt
           pip install -r requirements-dev.txt
       - name: Running flake8
-        run: flake8 vstools
+        run: |
+          flake8 vstools
+          flake8 tests
       - name: Running mypy
-        if: steps.dependencies.outcome == 'success'
-        run: mypy vstools
\ No newline at end of file
+        continue-on-error: true
```

Comment:
Seems there's a lot of stuff in the codebase that mypy is yelling about... not sure if it necessarily makes sense to fix it all in this PR, it could get quite large.

Reply:
then ignore and revert the changes

Reply:
With `continue-on-error: true` seems to ignore it and go on to the tests. Which seems like they're failing because of some missing pip dependencies, I'll have to get those fixed up.